lib/turn.rb

```diff
@@ -1,29 +1,122 @@
-require 'colorize'
+# require 'colorize'
 
 class Turn
 
-    attr_reader :player_board,
-                :cpu_board,
-                :cpu_cruiser,
-                :cpu_submarine,
-                :player_cruiser,
-                :player_submarine
-
-    def initialize 
-        @player_board = Board.new
-        @cpu_board = Board.new
-        @cpu_cruiser = Ship.new("Cruiser", 3)
-        @cpu_submarine = Ship.new("Submarine", 2)
-        @player_cruiser = Ship.new("Cruiser", 3)
-        @player_submarine = Ship.new("Submarine", 2)
-    end
+    # attr_reader :player_board,
+    #             :cpu_board,
+    #             :cpu_cruiser,
+    #             :cpu_submarine,
+    #             :player_cruiser,
+    #             :player_submarine
 
-    def cpu_place_ships
+    # def initialize 
+    #     @player_board = Board.new
+    #     @cpu_board = Board.new
+    #     @cpu_cruiser = Ship.new("Cruiser", 3)
+    #     @cpu_submarine = Ship.new("Submarine", 2)
+    #     @player_cruiser = Ship.new("Cruiser", 3)
+    #     @player_submarine = Ship.new("Submarine", 2)
+    # end
 
-    end
+    # def cpu_place_ships
+
+    # end
     
-    def player_place_ships
+    # def player_place_ships
+        
+    # end
+end
+
+
+    def initialize
+      @player_board = Board.new
+      @cpu_board = Board.new
+    end
+  
+    def start_game
+            puts "Welcome to Battleship!"
         
+      
```

Comment:
Let's chat about this tomorrow! I put game start in the gameplay file to try to "encapsulate" the classes so they focus just on what their classes would logically handle. Which now makes me wonder if ship placement should also be handled by the gameplay file...